require multiple BCD tables by specifying `browser-compat` as an array (#20051)
mdn/content: files/en-us/mdn/writing_guidelines/page_structures/compatibility_tables/index.md

The compatibility data query should be specified in the page front-matter using the `browser-compat` key.
For example, for the {{domxref("AbortController")}} this would be added as shown below:

> ```yaml

Reviewer: ```suggestion
```md
```

The compatibility and specification tables corresponding to the key are then automatically rendered in place of the `\{{Compat}}` and `\{{Specifications}}` macros in the source.

> **Note:** If multiple compatibility/specification tables are required on the same page, you can specify the value of `browser-compat` as an array.
> For example, for the {{domxref("Channel Messaging API")}} this would be added as shown below:
>

> > ```yaml

Reviewer: ```suggestion
> ```md
```

> > **Note:** If multiple compatibility/specification tables are required on the same page, you can specify the value of `browser-compat` as an array.
> > For example, for the {{domxref("Channel Messaging API")}} this would be added as shown below:
> >
> > ```md
> > ---
> > title: Channel Messaging API
> > slug: Web/API/Channel_Messaging_API
> >
> > …
> >
> > browser-compat:
> >   - api.MessageChannel
> >   - api.MessagePort
> > ---
> > ```

Reviewer: Maybe de-note this?

````suggestion
If multiple compatibility/specification tables are required on the same page, you can specify the value of `browser-compat` as an array.

For example, for the [Channel Messaging API](/en-US/docs/Web/API/Channel_Messaging_API) this would be added as shown below:

```md
---
title: Channel Messaging API
slug: Web/API/Channel_Messaging_API
# …
browser-compat:
  - api.MessageChannel
  - api.MessagePort
---
```
````

Author: There are two examples of BCD and Specifications tables below, and they are corresponding to "single" value of `browser-compat` metioned above. Using note card may be better?

Reviewer 2: I think the suggestion is good, the note is not necessary in this case.

Author: I've landed this.
